src/vivarium_nih_us_cvd/components/population.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+import pandas as pd
+from vivarium.framework.engine import Builder
+from vivarium.framework.population import SimulantData
+from vivarium_public_health.population import BasePopulation
+
+from vivarium_nih_us_cvd.constants import data_keys
+
+
+class EvenlyDistributedPopulation(BasePopulation):

```

Comment:
I'm assuming this fixed the 0-paf bug?

Reply:
It is at least intended to fix the zero PAF issue. We won't know until we try it.